src/winetricks

```diff
@@ -10142,6 +10142,145 @@ load_ie8_kb2936068()
+    #
+    # admparse.dll  advpack.dll  advpack.dll.mui  browseui.dll  corpol.dll  dxtmsft.dll  dxtrans.dll
+    # hmmapi.dll  html.iec  icardie.dll  idndl.dll  ie4uinit.exe  ie4uinit.exe.mui  ieakeng.dll
+    # ieaksie.dll  ieakui.dll  ieapfltr.dat  ieapfltr.dll  iedkcs32.dll  iedkcs32.dll.mui  ieframe.dll
+    # ieframe.dll.mui  iepeers.dll  ieproxy.dll  iernonce.dll  iertutil.dll  iesetup.dll  ieudinit.exe
+    # ieui.dll  imgutil.dll  inetcpl.cpl  inseng.dll  jscript.dll  jsproxy.dll  licmgr10.dll
+    # msdbg2.dll  msfeedsbs.dll  msfeeds.dll  msfeedssync.exe  mshta.exe  mshta.exe.mui  mshtml.dll
+    # mshtmled.dll  mshtmler.dll  mshtml.tlb  msls31.dll  msrating.dll  msrating.dll.mui  mstime.dll
+    # nlsdl.dll  normaliz.dll  occache.dll  pngfilt.dll  shdocvw.dll  shlwapi.dll  spmsg.dll
+    # spupdsvc.exe  sqmapi.dll  tdc.ocx  url.dll  urlmon.dll  vbscript.dll  verdcbc.tmp  VGX.dll
+    # w_try_regsvr64()webcheck.dll  wieframe.dll  WinFXDocObj.exe  wininet.dll  xmllite.dll
+    #
+    # + bunch of mui files in en-US
+
+    # Somehow it fails for me when WINEARCH=win32 is set; I`m guessing the installer doesn`t support this (?) (Bad exe)
+    w_package_unsupported_win32
+
+    w_set_winver win2k3
+
+    # Unregister Wine IE
+    if grep -q -i "wine placeholder" "$W_PROGRAMS_X86_UNIX/Internet Explorer/iexplore.exe"; then
+        w_override_dlls builtin iexplore.exe
+        w_try "$WINE" iexplore -unregserver
+    fi
+
+    # FIXME: Do we need more? Or less....
+    # Change the override to the native so we are sure we use and register them
+    w_override_dlls native,builtin ieframe ieproxy jscript mshtml shdocvw shlwapi urlmon vbscript wininet xmllite
+
+    # Bundled updspapi cannot work on Wine
+    w_override_dlls builtin updspapi
+
+    #    Can we also use sed to replace iexplore.exe's version string???? No idea
+    #    Removing the fake iexplore.exe's doesn`t work. Installer still complains a newer version
+    #    of IE is installed and refuses to run. Replacing fake iexplore.exe with one that
+    #    has a lower version string works.
+    #    Fake iexplore.exe generated with https://bugs.winehq.org/attachment.cgi?id=65708&action=diff
+    #
+    #    Get fake iexplore with low version string and copy it to approriate places
+
+    # TODO: checksum
+    if w_workaround_wine_bug 34048 'Exporting fake iexplore.exe to pass the version check'; then
+        w_download_to "$W_CACHE/$W_PACKAGE" https://bugs.winehq.org/attachment.cgi?id=65709 318ca190ffcc788e822fd7e0616ce44d9388c08261a3f1c19517e65a4f5c0732 "patch-65709"  #32-bit
+        w_download_to "$W_CACHE/$W_PACKAGE" https://bugs.winehq.org/attachment.cgi?id=65710 65eaae7b2165fe4415bfb123eb179cbdd24d77ce41fe62c9ea5850f1ca83d0b6 "patch-65710" #64-bit
+
+        w_try cp -f "$W_CACHE/$W_PACKAGE/patch-65709" "$WINEPREFIX/drive_c/windows/syswow64/iexplore.exe"
+        w_try cp -f "$W_CACHE/$W_PACKAGE/patch-65710" "$WINEPREFIX/drive_c/windows/system32/iexplore.exe"
```

Comment:
Variables preferred.

Reply:
Somehow there seems to be a problem with the expansion of $W_PROGRAMS_UNIX;

For another user it still expanded to Program Files x86, i don`t know why. For safety it is still hard coded in current version of patch.

Do you know why it does expand wrong?

Reply:
It depends on if a 32-bit or 64-bit process was invoked:
```
$ rm -rf ~/.wine; WINE=/usr/lib/wine-vanilla-4.0.1/bin/wine64 winetricks -q ie8
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20190912-next - sha256sum: 2e8bc060761ef22f41a50e5f8ad25e48f595f602886227683e097f90e7b81629 with wine-4.0.1 and WINEARCH=win64
Executing w_do_call ie8
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_ie8 
/home/austin/.wine/dosdevices/c:/Program Files

$ rm -rf ~/.wine; WINE=/usr/lib/wine-vanilla-4.0.1/bin/wine winetricks -q ie8
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Using winetricks 20190912-next - sha256sum: 2e8bc060761ef22f41a50e5f8ad25e48f595f602886227683e097f90e7b81629 with wine-4.0.1 and WINEARCH=win64
Executing w_do_call ie8
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
/home/austin/.wine/dosdevices/c:/Program Files (x86)
```

Hardcoding it will break it for users with a non-english locale, where Program Files has a different name.